src/components/transactions/Repay/CollateralRepayModalContent.tsx

```diff
@@ -247,8 +325,14 @@ export function CollateralRepayModalContent({
         />
       </Box>
       <AssetInput
-        value={swapVariant === 'exactOut' ? inputAmount : tokenToRepayWithBalance}
-        usdValue={inputAmountUSD}
+        value={
+          swapVariant === 'exactOut'
```

Comment:
I think there's a scenario where the variant could get set to `exactIn` and then if the user decided to switch to repaying with a tokens, it would default to the users aToken balance.

Reply:
nice catch, this commit should resolve this f91964e08f7ddc47b9f1bbbfd449180f26ae5b73